hack/update_strimzi.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
TBH I prefer to avoid committing the strimzi setup and always download from GH like on master, is that possible or is it causing issues?

Reply:
We need to do the dance to change the cluster suffix manually.

Reply:
Also this makes our builds reproducible.

Reply:
Ok I just checked eventing-kafka, they have the strimzi setup file committed, so ok let's keep it here. Can you add a comment explaining that this script updates the strimzi file we use in builds?